extensions/roc-plugin-repo/src/semver/conventional-changelog-angular-transformer.js

```diff
@@ -0,0 +1,73 @@
+/**
+ Code taken from conventional-changelog-angular
+*/
+
+/* eslint-disable */
+export default function(commit, context) {
+  var discard = true;
+  var issues = [];
+
+  commit.notes.forEach(function(note) {
+    note.title = 'BREAKING CHANGES';
+    discard = false;
+  });
+
+  if (commit.type === 'feat') {
+    commit.type = 'Features';
+  } else if (commit.type === 'fix') {
+    commit.type = 'Bug Fixes';
+  } else if (commit.type === 'perf') {
+    commit.type = 'Performance Improvements';
+  } else if (commit.type === 'revert') {
+    commit.type = 'Reverts';
+  } else if (discard) {
+    return;
+  } else if (commit.type === 'docs') {
+    commit.type = 'Documentation';
+  } else if (commit.type === 'style') {
+    commit.type = 'Styles';
+  } else if (commit.type === 'refactor') {
+    commit.type = 'Code Refactoring';
+  } else if (commit.type === 'test') {
+    commit.type = 'Tests';
+  } else if (commit.type === 'chore') {
+    commit.type = 'Chores';
```

Comment:
I will not change this code since I want it to map as much as possible to the actual implementation since this should only be temporary until a new version of the package has been published.